### DISCUSSION TITLE
Increase the webview loading speed.

### DISCUSSION THREAD

**Discussion by Harikarthyk** [react-native-webview/react-native-webview#2780] [Q&A]
Hello,

I am trying to render a list of URLs in the webview, and for the first time, everything takes up to 8 to 15 seconds, which is very slow.

Is there any possible way to reduce the rendering speed?

NOTE: (List of URL's are dynamic and can also be many).

Can you please suggest me a solution to render things quickly like a browsers ?

**Comment by jordanjoecooper** [comment]
Depending on what you are rendering (and how)  this might be helpful: https://reactnative.dev/docs/optimizing-flatlist-configuration

Beyond that you could consider only rendering a limited number of items in the list initially and then use pagination (if its a really really big list)
